x-pack/plugins/security_solution/public/hosts/pages/details/index.tsx

```diff
@@ -187,6 +188,11 @@ const HostDetailsComponent: React.FC<HostDetailsProps> = ({ detailName, hostDeta
                   />
                 }
                 title={detailName}
+                rightSideItems={[
+                  hostOverview.endpoint?.fleetAgentId && (
+                    <ResponderActionButton endpointId={hostOverview.endpoint?.fleetAgentId} />
+                  ),
+                ]}
```

Comment:
@ashokaditya my thinking was to have the button take care of checking for permissions on it's own for reusability.  Otherwise when we use it in other places, we'll have to repeat the permissions check